charts/internal/seed-controlplane/charts/csi-driver-controller/templates/csi-driver-controller-deployment.yaml

```diff
@@ -76,7 +76,7 @@ spec:
         - name: socket-dir
           mountPath: {{ .Values.socketPath }}
 
-      - name: csi-provisioner
+      - name: aws-csi-provisioner
```

Comment:
I don't get why you change the container names? There is nothing aws specific in this container to call it `aws-csi-provisioner`.

Reply:
Because when the seed contains multiple provider-extensions, in the fluent-bit Configmap will be added the same rewrite_tag filter: https://github.com/gardener/gardener-extension-provider-aws/pull/226/files#diff-019be355db9cbaece5133ea2a8f44876159eb1b7d2a119906d1c9b3d3c09a9c3R31
multiple times

Reply:
It is the same for ccm/mcm

Reply:
Yeah but ccm has only 1 container it is cloud-provider specific and you can name it `{provider}-cloud-controller-manager`. For CSI we have 1 cloudprovider specific container and the other 5 sidecar containers are not.

Reply:
IMO for the CSI case it is better to rename the deployment to `{provider}-csi-driver-controller`. Calling the external-provisioner `aws-csi-provisioner` sounds misleading.

Reply:
@ialidzhikov if we do this then I suggest to also rename the CCM/MCM deployments so that it's consistent. WDYT?

Reply:
Sounds good to me.

Reply:
With the out-of-tree of MCM, it already have the [provider name](https://github.com/gardener/gardener-extension-provider-gcp/blob/89f8007cdfa09cb5939eb000f399fc1eb1a0d372/charts/internal/machine-controller-manager/seed/templates/deployment.yaml#L42-L71) in its containers:
- gcp-machine-controller-manager
- controller-manager-provider-gcp

Maybe the general controller can then remove the provider name from the container name?